tests/create.test.ts

```diff
@@ -0,0 +1,21 @@
+import { PGlite } from "@electric-sql/pglite";
```

Comment:
Whoa nice; I tried to get our normal tests to work with this, but the exposed interface wasn't working with what zapatos calls for transactions.

Reply:
Yeah, I had to write a shim client that wrapped calls to the `PGlite` db. I actually don't think the runtime will be faster after playing with it, but it's a lot easier to use for testing -- e.g., the big reason I never did this before was setting up the test infra would have been a massive pain.

Reply:
You mean Docker?

Reply:
Yep, Docker, testcontainers etc